### PR TITLE
Feature/customer info

### PR DESCRIPTION
**Pull request by HannesOster**
added the feature to add a customer info to the customer data

### CODE REVIEW

File: components/CustomerInfo/CustomerInfo.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+} from "./styles";
+
+import { GreenButton, RedButton } from "../Buttons/styles";
+import { Form } from "../../pagestyles/styles";
+
+export default function CustomerInfo({ customer, id }) {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const { mutate } = useSWR("/api");
```
Comment (Kheirah) [inline components/CustomerInfo/CustomerInfo.js]: The mutate function only updates your view when you specify the exact same api endpoint as you use for the PATCH request. Currently, your page does not update the change. It only updates when you click outside the browser window and back in. Instead, what you want is to immediately react to the database change.

```suggestion
  const { mutate } = useSWR(`/api/${id}`);
```

File: components/CustomerInfo/CustomerInfo.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            Kundeninfo bearbeiten
+          </button>
+        </>
+      )}
+      <ReactModal
+        isOpen={isModalOpen}
+        onRequestClose={() => setIsModalOpen(false)}
+        contentLabel="Kundeninfo bearbeiten"
+      >
+        <CustomerInfoForm onSubmit={(event) => editCustomerInfo(event, id)}>
```
Comment (Kheirah) [inline components/CustomerInfo/CustomerInfo.js]: You don't need to pass the id here. You're not using it in your handle function anyway.

**Review by Kheirah — CHANGES_REQUESTED** [review]
Good work!

Have a look at the mutate function and see if my suggested changes make sense. I haven't tested it, so make sure it works when you try it out.

File: components/CustomerInfo/CustomerInfo.js (hunk 1 above)
Comment (HannesOster) [reply components/CustomerInfo/CustomerInfo.js]: Thanks! I was looking for the reason, why the customerinfo isnt displayed before refreshing

**Review by ekat-br — APPROVED** [review]

**Pull request merged**